pulse2percept/models/base.py

```diff
@@ -417,6 +418,53 @@ def plot(self, use_dva=False, style='hull', autoscale=True, ax=None,
             ax.set_ylabel('y (microns)')
         return ax
 
+    def __eq__(self, other):
```

Comment:
It looks like both __eq__ and __deepcopy__ are almost the same for both SpatialModel and Temporal model. Both inherit from BaseModel, so would it be possible to make changes there instead?